crypto/aes/aes_ige.c

```diff
@@ -38,15 +39,31 @@ void AES_ige_encrypt(const unsigned char *in, unsigned char *out,
                      size_t length, const AES_KEY *key,
                      unsigned char *ivec, const int enc)
 {
+    /* Ignore the return value since we are void */
+    int ret =AES_ige_encrypt_ex(in, out, length, key, ivec, enc);
+
+    assert(ret);
```

Comment:
It is, but works with me.  I think it's valuable to be able to see the call directly when the assertion fails.